snap/frontend/package.json

```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "@metamask/detect-provider": "^2.0.0",
     "next": "14.1.3",
+    "qrcode": "^1.5.3",
```

Comment:
Did we standardize on a  QR code?

Reply:
I don't think we have **official** standardize QR code, but we did have the library [symbol-qr](https://github.com/symbol/qr-library) (old library, used by our mobile / desktop wallet app)